packages/storybook-html/src/components/accordion/accordion-subtitle.html

```diff
@@ -0,0 +1,25 @@
+<div class="uq-accordion" aria-label="Accordion button group" role="presentation">
+    <div class="uq-accordion__item">
+    <button class="uq-accordion__toggle" aria-controls="content-1" aria-expanded="false" id="accordion-title-1">
+        Title 1
+        <span class="uq-accordion__subtitle">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum auctor eros dui, vitae iaculis leo ornare eget.</span>
+    </button>
+    <div class="uq-accordion__content" role="region" aria-hidden="true" id="content-1" aria-labelledby="accordion-title-1">
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum auctor eros dui, vitae iaculis leo ornare eget. Etiam tristique lobortis ligula non mattis. Ut dapibus libero neque, vel tincidunt nunc tempus eget.</p>
+    </div>
+    </div>
+    <div class="uq-accordion__item">
+    <button class="uq-accordion__toggle" aria-controls="content-2" aria-expanded="false" id="accordion-title-2">Title 2</button>
```

Comment:
Should you put subtitles on the other 2 for this example?

Reply:
I looked it over again, as the style does not change regardless of the other variants it doesn't affect regression testing, so going to leave it as is